packages/amplify-nodejs-function-runtime-provider/src/utils/execute.ts

```diff
@@ -105,5 +105,5 @@ process.on('message', async options => {
       }),
     );
   }
-  exit(1);
+  exit(0);
```

Comment:
The described issue in the description of this PR looks correct to me. For the solution part, we might want to handle `process.exit` correctly for both success and failure cases. That being said this block should have `exit(0)` for success case and `exit(1)` for failure case as same as the mock IMHO.

Refs: https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-util-mock/src/utils/lambda/execute.ts#L96-L105

cc @edwardfoyle @jhockett

Reply:
Thank you for your comments and suggestions! 🙏 

I think your suggestion is correct and I've made a fix based on the link of Refs. 🛠️